lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
The problem with your example is that Ruby will blow up if you don't pass `id`. On the other hand `option` is a hash. And you could technically add anything you want in it.

```ruby
# works.
channel.history(something: 'hello')

# blows up
channel.history(until_attach: true)
```